src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -169,6 +172,22 @@ public void handleLoanUpdate(LoanDTO loan) {
     }
   }
 
+  @Override
+  public void handleRequestUpdate(RequestDTO requestDTO) {
+    var transaction = transactionRepository.fetchOneByRequestId(requestDTO.getId()).orElse(null);
+    if (transaction == null) {
+      return;
+    }
+
+    TransactionHold transactionItemHold = transaction.getHold();
+    if (!transactionItemHold.getFolioItemId().equals(requestDTO.getItemId())) {
```

Comment:
> mod-inn-reach should check if the request is associated with an active INN-Reach **Item Hold** transaction

Reply:
We check it here  var transaction = transactionRepository.fetchOneByRequestId(requestDTO.getId()).orElse(null);

Reply:
it doesnt check if transaction is Item Hold

Reply:
Fixed